privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/ui/PrivacyDashboardHybridViewModel.kt

```diff
@@ -52,20 +53,23 @@ class PrivacyDashboardHybridViewModel @Inject constructor(
     private val requestDataViewStateMapper: RequestDataViewStateMapper,
     private val protectionStatusViewStateMapper: ProtectionStatusViewStateMapper,
     private val privacyDashboardPayloadAdapter: PrivacyDashboardPayloadAdapter,
+    private val autoconsentStatusViewStateMapper: AutoconsentStatusViewStateMapper,
 ) : ViewModel() {
 
     private val command = Channel<Command>(1, DROP_OLDEST)
 
     sealed class Command {
         class LaunchReportBrokenSite(val data: BrokenSiteData) : Command()
         class OpenURL(val url: String) : Command()
+        class OpenSettings(val target: String) : Command()
```

Comment:
I would renamed this command to `OpenAutoconsentSettings` to avoid confusion with Settings root.